android/package.json

```diff
@@ -13,6 +13,7 @@
 	"maintainers": [
 		"Chris Barber <cbarber@appcelerator.com>"
 	],
+	"architectures": ["arm64-v8a", "armeabi-v7a", "x86"],
```

Comment:
Just for future reference, we should group all our custom settings under a common`titanium` key so we don't clutter the `package.json`

Reply:
Agree. But trying to not change too many things in a minor version. Will do for 7.1.0 in an own PR.